<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bookshop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableViewBooks">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>941</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNewBook">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;New Book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnDialogGenre">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Genre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeleteBook">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete &amp;Book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBuyBook">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>B&amp;uy Book</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>979</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
